cms/envs/common.py

```diff
@@ -1341,7 +1341,7 @@
         'text/csv',
         'text/pdf',
         'text/x-sh',
-        '\application/pdf\""',
+        '\"application/pdf\"',
```

Comment:
@MichaelRoytman I wasn't really sure about this, but it looked like a typo since I think the "a" was getting escaped not the quote.